staging/src/k8s.io/apiserver/pkg/audit/format.go

```diff
@@ -61,7 +61,7 @@ func EventString(ev *auditinternal.Event) string {
 	}
 
 	return fmt.Sprintf("%s AUDIT: id=%q stage=%q ip=%q method=%q user=%q groups=%q as=%q asgroups=%q user-agent=%q namespace=%q uri=%q response=\"%s\"",
-		ev.RequestReceivedTimestamp.Format(time.RFC3339Nano), ev.AuditID, ev.Stage, ip, ev.Verb, username, groups, asuser, asgroups, ev.UserAgent, namespace, ev.RequestURI, response)
+		ev.RequestReceivedTimestamp.UTC().Format(time.RFC3339Nano), ev.AuditID, ev.Stage, ip, ev.Verb, username, groups, asuser, asgroups, ev.UserAgent, namespace, ev.RequestURI, response)
```

Comment:
how is this method used? if it is in a log file that is also using local timestamps, local may be better than utc